CHANGELOG.md

```diff
@@ -8,6 +8,102 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+### Known Knowns
+
+- Relevant components and interfaces developed considering Gaia-X Trust Framework - 22.10
+- Connection to an external Clearing House service for the entire Catena-X ecosystem is required. The Gaia-X compliance service should generally be capable of recognizing all companies that are able to register with the Catena-X Portal.
+Conformity to [Asset Administration Shell (AAS) API v3.0](https://industrialdigitaltwin.org/wp-content/uploads/2023/04/IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf) and bugfix v3.0.1 is given. Fine granular access control introduced for optional use.
+- This Tractus-X release does not include a Managed Identity Wallet (MIW). The development team decided on a temporary bridging solution, based on a commercial application, to test and ship Release 24.08. Without implementing this interim solution, participants will not be able to fully utilize this Tractus-X release in consequence.
+- For E2E Testing, the [1.2.0](https://github.com/SAP/ssi-dim-middle-layer/releases/tag/v1.2.0) of the [SSI DIM Middle Layer](https://github.com/SAP/ssi-dim-middle-layer) was used - this component is FOSS but the currently used wallet (see previous item) is not.
+- The Tractus-X EDC has undergone significant changes in its Management API, communication with external services and internal APIs.
+- Documentation content (e.g. architecture, manuals etc) and folder structure have not explicitly been checked for this Release, as relevant TRGs are not yet in force. The vast majority of components are evolutions of previous Releases, with a thorough review undergone. This leads to a high confidence in the quality of existing documentation within the product repositories.
+- FOSS components developed and pre-tested to TRL6 with artificial test data only
+- No crosscheck of functionality in verification environment
+- No execution of Load-, Performance- or Penetration Tests
+- No assignment of Export Control Classification Numbers (ECCN) to FOSS components
+- GeoBlocking recommended for Operations (GBaaS)
+- Security concept (overall) recommended for Operations
+- Listed components in "Unchanged, untested" category were not verified in combination with the current Release package and are only compatible with Catena-X Standards as specified. Use at your own risk at this time.
+
+### Runtime
+
+The release has been tested
+
+- on [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes) versions: `1.28.9`
+- with [PostgreSQL](https://en.wikipedia.org/wiki/PostgreSQL) versions: `15`
+
+Note: Tractus-X EDC has been tested on both Postgresql versions: 15.x and 16.x
+
+The following violations of TRG 5.07 apply:
+
+- Golden Record Business Partner Number (BPN) Service operates on Postgresql version 16.x (not 15), but has been successfully tested for the current Release
```

Comment:
As mentioned above, not applicable anymore. BPDM is not in violation of TRG 5.07 anymore

Reply:
Following the [app dashboard](https://app.int.demo.catena-x.net/) a postgresql 16 version has been used on INT. Only tested versions are relevant.

I also don't see an EDC which might use the newer version

Certificate Management thus would be needed to be added as it uses a 14.x version

Reply:
@tom-rm-meyer-ISST Not sure if i did it correct, but i deleted the violation on bpdm side. Is there a need to change something related to postgres? What about country risk violation?

Reply:
Answer can be found [here](https://github.com/eclipse-tractusx/tractus-x-release/pull/34#discussion_r1701307857)